### PR TITLE
Remove 2 Templates from Solutions

### PR DESCRIPTION
**Pull request by Mastermind497**
No functionality has been changed, just the templates have been removed. The solutions in question are:

- Livestock Lineup (Bronze)
- Swapity Swap (Bronze)

### CODE REVIEW

**Comment by Mastermind497** [comment]
This references issue #806

File: solutions/USACO dec19-lineup.mdx [hunk 1]
```diff
@@ -25,239 +25,65 @@ path containing the cow to the answer.
...
-	// ex. try to read letter into int
-	if (sz(s)) setIn(s+".in"), setOut(s+".out"); // for USACO
-}
-
 // lambda expression - see https://usaco.guide/general/lambda
-vs cows = []() {
-	vs tmp{"Bessie", "Buttercup", "Belinda", "Beatrice", "Bella", "Blue", "Betsy", "Sue"};
-	sort(all(tmp)); // sort names lexicographically
-	return tmp;
+vector<string> cows = []() {
```
Comment (maggie-j-liu) [inline solutions/USACO dec19-lineup.mdx:29]: use tabs instead of spaces to indent

File: solutions/USACO dec19-lineup.mdx [hunk 2]
```diff
@@ -25,239 +25,65 @@ path containing the cow to the answer.
...
+    vector<string> tmp{"Bessie", "Buttercup", "Belinda", "Beatrice",
+                       "Bella",  "Blue",      "Betsy",   "Sue"};
```
Comment (maggie-j-liu) [inline solutions/USACO dec19-lineup.mdx:31]:
```suggestion
                       "Bella", "Blue", "Betsy", "Sue"};
```

File: solutions/USACO dec19-lineup.mdx (hunk 1 above)
Comment (Mastermind497) [reply solutions/USACO dec19-lineup.mdx:29]: oops, I usually do but I used a formatter, so that must have changed it. Let me fix it real quick

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

**Review by maggie-j-liu — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**